HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -576,56 +576,194 @@ def self.apply_ground_to_air_heat_pump(model, runner, weather, heat_pump, hvac_s
+        coeff: hp_ap.cool_cap_curve_spec[0]
+      )
+      clg_sens_cap_curve = Model.add_curve_quint_linear(
+        model,
+        name: "#{obj_name} clg sens cap curve",
+        coeff: hp_ap.cool_sh_curve_spec[0]
+      )
+      clg_power_curve = Model.add_curve_quad_linear(
+        model,
+        name: "#{obj_name} clg power curve",
+        coeff: hp_ap.cool_power_curve_spec[0]
+      )
+      clg_coil = OpenStudio::Model::CoilCoolingWaterToAirHeatPumpEquationFit.new(model, clg_total_cap_curve, clg_sens_cap_curve, clg_power_curve)
+      clg_coil.setRatedCoolingCoefficientofPerformance(hp_ap.cool_rated_cops[0])
+      clg_coil.setNominalTimeforCondensateRemovaltoBegin(1000)
+      clg_coil.setRatioofInitialMoistureEvaporationRateandSteadyStateLatentCapacity(1.5)
+      clg_coil.setRatedAirFlowRate(UnitConversions.convert(clg_cfm_rated, 'cfm', 'm^3/s'))
+      clg_coil.setRatedWaterFlowRate(UnitConversions.convert(geothermal_loop.loop_flow, 'gal/min', 'm^3/s'))
+      clg_coil.setRatedEnteringWaterTemperature(UnitConversions.convert(80, 'F', 'C'))
+      clg_coil.setRatedEnteringAirDryBulbTemperature(UnitConversions.convert(80, 'F', 'C'))
+      clg_coil.setRatedEnteringAirWetBulbTemperature(UnitConversions.convert(67, 'F', 'C'))
+      clg_coil.setRatedTotalCoolingCapacity(UnitConversions.convert(heat_pump.cooling_capacity, 'Btu/hr', 'W'))
+      clg_coil.setRatedSensibleCoolingCapacity(UnitConversions.convert(hp_ap.cooling_capacity_sensible, 'Btu/hr', 'W'))
+      # Heating Coil
+      htg_cap_curve = Model.add_curve_quad_linear(
+        model,
+        name: "#{obj_name} htg cap curve",
+        coeff: hp_ap.heat_cap_curve_spec[0]
+      )
+      htg_power_curve = Model.add_curve_quad_linear(
+        model,
+        name: "#{obj_name} htg power curve",
+        coeff: hp_ap.heat_power_curve_spec[0]
+      )
+      htg_coil = OpenStudio::Model::CoilHeatingWaterToAirHeatPumpEquationFit.new(model, htg_cap_curve, htg_power_curve)
+      htg_coil.setName(obj_name + ' htg coil')
+      htg_coil.setRatedHeatingCoefficientofPerformance(hp_ap.heat_rated_cops[0])
+      htg_coil.setRatedAirFlowRate(UnitConversions.convert(htg_cfm_rated, 'cfm', 'm^3/s'))
+      htg_coil.setRatedWaterFlowRate(UnitConversions.convert(geothermal_loop.loop_flow, 'gal/min', 'm^3/s'))
+      htg_coil.setRatedEnteringWaterTemperature(UnitConversions.convert(60, 'F', 'C'))
+      htg_coil.setRatedEnteringAirDryBulbTemperature(UnitConversions.convert(70, 'F', 'C'))
+      htg_coil.setRatedHeatingCapacity(UnitConversions.convert(heat_pump.heating_capacity, 'Btu/hr', 'W'))
+    else
+      num_speeds = hp_ap.cool_capacity_ratios.size
+      # TODO: Curve placeholder
+      plf_fplr_curve = Model.add_curve_quadratic(
+        model,
+        name: 'Cool-PLF-fPLR',
+        coeff: [1, 0, 0],
+        min_x: 0, max_x: 1, min_y: 0.7, max_y: 1
+      )
+      clg_coil = OpenStudio::Model::CoilCoolingWaterToAirHeatPumpVariableSpeedEquationFit.new(model, plf_fplr_curve)
```

Comment:
Be sure to audit the code for references to e.g. `CoilCoolingWaterToAirHeatPumpEquationFit`. I'm seeing at least two place in our code (installation quality EMS program and desuperheaters) that presumably need to be updated.

Reply:
We talked about desuperheaters, the variable speed coil type is not yet supported by E+ to be connected with desuperheater, so we may have to temporarily ignore that feature (and document that only single speed gshp is currently supported).